app/models/mixins/assignment_mixin.rb

```diff
@@ -81,17 +81,25 @@ def remove_all_assigned_tos(cat = nil)
   delegate :namespace, :to => :class
 
   module ClassMethods
+    # get a mapping of alert_sets and the tags they are assigned
+    #
+    # the namespace is removed from the front of the alert_set's tag
+    #   alert_set will have:
```

Comment:
this is also used for callbacks.

It is very abstract. I'll reword to turn into an example